apps/notifications/src/panel/templates/shortcuts-popover.jsx

```diff
@@ -0,0 +1,154 @@
+import { Popover } from '@automattic/components';
+import { isWithinBreakpoint, MOBILE_BREAKPOINT } from '@automattic/viewport';
+import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
+import { useRef } from 'react';
+import { connect } from 'react-redux';
+import actions from '../state/actions';
+import getIsPanelOpen from '../state/selectors/get-is-panel-open';
+import getIsShortcutsPopoverOpen from '../state/selectors/get-is-shortcuts-popover-open';
+import getKeyboardShortcutsEnabled from '../state/selectors/get-keyboard-shortcuts-enabled';
+import { getSelectedNoteId } from '../state/selectors/get-selected-note-id';
+import HotkeyContainer from './container-hotkey';
+import Gridicon from './gridicons';
+
+export const ShortcutsPopover = ( {
+	closeShortcutsPopover,
+	toggleShortcutsPopover,
+	isPanelOpen,
+	isShortcutsPopoverOpen,
+	isMobile,
+} ) => {
+	const translate = useTranslate();
+
+	// create context for the keyboard shortcuts popover icon
+	const popoverAnchorRef = useRef();
+	const spanRef = useRef();
+
+	// This function renders a list of keyboard shortcuts
+	const renderShortcutsPopover = () => {
+		return (
+			<Popover
+				onClose={ closeShortcutsPopover }
+				isVisible={ isPanelOpen && isShortcutsPopoverOpen }
+				context={ popoverAnchorRef.current }
+				ignoreContext={ spanRef.current }
+				position="bottom left"
+				className="wpnc__keyboard-shortcuts-popover"
+			>
+				{ /* Ignore linter rules since we just want to prevent the notifications panel from closing if user clicks inside the popover... */ }
+				{ /* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */ }
+				<div
+					onClick={ ( event ) => {
+						event.stopPropagation();
+						event.preventDefault();
+					} }
+				>
```

Comment:
I also added this div to catch clicks since previously if we clicked inside the popover, it would close the entire notifications panel as well which is a bit jarring.

Reply:
That's a nice catch!